examples/openai-lander/evolve.py

```diff
@@ -211,14 +212,15 @@ def run():
                     # determine the best action given the current state.
                     votes = np.zeros((4,))
                     for n in best_networks:
-                        output = n.activate(observation)
+                        output = n.activate(observation_init_vals)
                         votes[np.argmax(output)] += 1
 
                     best_action = np.argmax(votes)
-                    observation, reward, done, info = env.step(best_action)
+                    # Note: done has been deprecated.
+                    observation, reward, terminated, done, info = env.step(best_action)
                     score += reward
                     env.render()
-                    if done:
+                    if terminated:
```

Comment:
I would use the truncated state since it seems more to behave like the old "done". 
According to the docstring truncated means:
```
truncated (bool): whether a truncation condition outside the scope of the MDP is satisfied.
               Typically a timelimit, but could also be used to indicate agent physically going out of bounds.
               Can be used to end the episode prematurely before a `terminal state` is reached.
```

Reply:
It looks like the correct action here would have been `if terminated or truncated`.